### PR TITLE
bluetooth off banner text & intro spacing

### PR DESCRIPTION
**Pull request by ckiss**
[ref](https://github.com/project-vagabond/covid-green-app/issues/418)
[ref1](https://github.com/project-vagabond/covid-green-app/issues/417)

### CODE REVIEW

File: src/components/views/dashboard.tsx [hunk 1]
```diff
@@ -76,30 +84,32 @@ export const Dashboard: FC<any> = ({navigation}) => {
...
         </>
       )}
-      {(!exposure.enabled ||
-        exposure.status.state !== StatusState.active ||
-        exposure.permissions.notifications.status !==
-          PermissionStatus.Allowed) && (
+      {(!enabled ||
+        status.state !== StatusState.active ||
+        permissions.notifications.status !== PermissionStatus.Allowed ||
+        bluetoothOff) && (
```
Comment (floridemai) [inline src/components/views/dashboard.tsx]: No need to include `bluetoothOff` here. It's implied if ENS is not active. Just pass `status.type?.indexOf(StatusType.bluetooth) !== -1` to `AlertInformation` component.

File: src/components/views/onboarding/styles.tsx [hunk 2]
```diff
@@ -4,8 +4,7 @@ import {text, colors, shadows} from 'theme';
 
 export const styles = StyleSheet.create({
   title: {
-    ...text.largeBold,
-    marginBottom: 20
+    ...text.largeBold
```
Comment (floridemai) [inline src/components/views/onboarding/styles.tsx]: This applies to other screens as well (permissions?). Is that correct?

File: src/components/views/onboarding/styles.tsx [hunk 3]
```diff
@@ -24,12 +23,11 @@ export const styles = StyleSheet.create({
   },
   groupOfPeople: {
     marginTop: -14,
-    // backgroundColor: colors.background,
     alignSelf: 'flex-start'
   },
   list: {
     flexDirection: 'row',
-    marginBottom: 16
+    marginBottom: 12
```
Comment (floridemai) [inline src/components/views/onboarding/styles.tsx]: Same: applied in multiple places? Is that correct? If this is fo the list that have a left border... is 12 the right values? Seems like it should be smaller? Designs looks like 2 (maybe 4?)

File: src/components/molecules/alert-information.tsx [hunk 4]
```diff
@@ -5,35 +5,42 @@ import {useNavigation} from '@react-navigation/native';
...
+  return (
+    <Card
+      ref={ref}
+      padding={{h: 10, r: 16}}
+      onPress={() => navigation.navigate(ScreenNames.MyCovidAlerts)}
+      type="info"
+      arrowColor={colors.black}>
+      <View style={styles.row}>
+        {bluetoothOff ? (
+          <AppIcons.BluetoothOff width={40} height={30} color={colors.text} />
```
Comment (floridemai) [inline src/components/molecules/alert-information.tsx:31]: 40x30 vs 40x40 (just checking as viewBox is 0 0 23 30)? Maybe ask @helloantoine for the correct size in both cases (if you didn't already)

File: src/components/molecules/alert-information.tsx [hunk 5]
```diff
@@ -5,35 +5,42 @@ import {useNavigation} from '@react-navigation/native';
...
+        ) : (
           <StateIcons.ExposureUnset
             width={40}
             height={40}
-            color={colors.black}
+            color={colors.text}
```
Comment (floridemai) [inline src/components/molecules/alert-information.tsx:36]: Why text and not black?

File: src/components/molecules/alert-information.tsx [hunk 6]
```diff
@@ -5,35 +5,42 @@ import {useNavigation} from '@react-navigation/native';
...
-          <Text style={styles.notice}>
-            {t('alertsUnavailablePrompt:title')}
-          </Text>
-        </View>
-      </Card>
-    );
-  }
-);
+        )}
+        {bluetoothOff ? <Text style={styles.notice}>{t('alertsUnavailablePrompt:bluetoothOff')}</Text> : <Text style={styles.notice}>{t('alertsUnavailablePrompt:title')}</Text>}
```
Comment (floridemai) [inline src/components/molecules/alert-information.tsx]: Seems like only the text is different. I'd move the condition "inside"

**Review by floridemai — COMMENTED** [review]
Why all the svg updates?
Can I ask you to also add the following: search for usages of "infoCard" (the Learn how to keep other safe) and replaced `largeBold` with `defaultBold`? Thanks!

File: src/components/views/onboarding/styles.tsx (hunk 2 above)
Comment (ckiss) [reply src/components/views/onboarding/styles.tsx]: I will check. maybe we should not have these shared styles...

File: src/components/views/onboarding/styles.tsx (hunk 3 above)
Comment (ckiss) [reply src/components/views/onboarding/styles.tsx]: same

File: src/components/molecules/alert-information.tsx (hunk 4 above)
Comment (ckiss) [reply src/components/molecules/alert-information.tsx:31]: @helloantoine can you please confirm the size for this icon?

File: src/components/molecules/alert-information.tsx (hunk 5 above)
Comment (ckiss) [reply src/components/molecules/alert-information.tsx:36]: for consistency. that's what we use for the other orange card.

**Review by floridemai — APPROVED** [review]

**Pull request merged**